### PR TITLE
Updating to kernel of CentOS 7.5 + Adding shutdown to inaugruator for qcow build

### PR DESCRIPTION
**Pull request by susyo**

### CODE REVIEW

**Comment by susyo** [comment]
@rom-stratoscale now its fixed

File: docker/build-inaugurator.dockerfile [hunk 1]
```diff
@@ -47,7 +44,7 @@ RUN pip install pep8 pika>=0.10.0
 RUN sed -i -e "s/Defaults    requiretty.*/ #Defaults    requiretty/g" /etc/sudoers
 
 # Install busybox with a Fedora RPM since there's no such package for Centos 7
-RUN curl ftp://195.220.108.108/linux/fedora/linux/releases/25/Everything/x86_64/os/Packages/b/busybox-1.22.1-5.fc24.x86_64.rpm -o temp && \
+RUN curl ftp://195.220.108.108/linux/fedora/linux/releases/28/Everything/x86_64/os/Packages/b/busybox-1.26.2-3.fc27.x86_64.rpm -o temp && \
```
Comment (rom-stratoscale) [inline docker/build-inaugurator.dockerfile]: @susyo this is the latest?

File: sh/list_storage_and_network_driver_names.sh [hunk 2]
```diff
@@ -3,4 +3,4 @@
 BLACKLIST='-e bfa -e csiostor -e cxgb4 -e bna -e aic94xx'
 KERNEL_VERSION=$1
 set -e
-find /lib/modules/$KERNEL_VERSION/kernel/drivers/net/ethernet /lib/modules/$KERNEL_VERSION/kernel/drivers/scsi -type f -printf '%f\n' | sed 's/\.ko$//' | grep -v $BLACKLIST
+find /lib/modules/$KERNEL_VERSION/kernel/drivers/net/ethernet /lib/modules/$KERNEL_VERSION/kernel/drivers/scsi /lib/modules/$KERNEL_VERSION/kernel/drivers/nvme /lib/modules/$KERNEL_VERSION/kernel/drivers/message -type f -printf '%f\n' | sed 's/\.ko$//' | sed 's/\.ko.xz$//' | grep -v $BLACKLIST
```
Comment (rom-stratoscale) [inline sh/list_storage_and_network_driver_names.sh]: @susyo what is the reason for this change? Seems legit jsut want to understand why

**Comment by rom-stratoscale** [comment]
Jira tickets please

File: docker/build-inaugurator.dockerfile [hunk 3]
```diff
@@ -27,9 +27,6 @@ RUN yum install -y \
     yum -y clean all
 
 
-RUN wget http://linuxsoft.cern.ch/cern/centos/7/updates/x86_64/Packages/kernel-3.10.0-514.6.1.el7.x86_64.rpm
```
Comment (rom-stratoscale) [inline docker/build-inaugurator.dockerfile]: @susyo not sure that u should remove it. What does exist in the container? I cannot see any kernel rpm installed there

File: sh/list_storage_and_network_driver_names.sh (hunk 2 above)
Comment (susyo) [reply sh/list_storage_and_network_driver_names.sh]: adding more storage driverss and network drivers  that are not in the above paths

Comment (susyo) [reply sh/list_storage_and_network_driver_names.sh]: adding more storage and network drivers that are not in the above paths

File: docker/build-inaugurator.dockerfile (hunk 3 above)
Comment (roeigo-stratoscale) [reply docker/build-inaugurator.dockerfile]: Does it have to take packages from the internet?
